### PR TITLE
Update domain model for determination of means

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link (if applicable) ###
[ROC-4346](https://tools.hmcts.net/jira/browse/ROC-4346)

### CODE REVIEW

File: domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/RepaymentPlan.java [hunk 1]
```diff
@@ -14,8 +15,8 @@
 
 import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
 
+@Getter
 @EqualsAndHashCode
-@JsonIgnoreProperties(value = {"firstPayment"})
```
[the remark this answers is not in the document]
Comment (sabahirfan) [reply domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/RepaymentPlan.java]: I think someone added by mistake, there is no such property.

File: src/main/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceService.java [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+            .repaymentPlan(acceptedPaymentIntention.getRepaymentPlan().orElse(null))
+            .payBySetDate(acceptedPaymentIntention.getPaymentDate().orElse(null))
+            .build();
+
+        this.countyCourtJudgmentService.save(
+            claim.getSubmitterId(),
+            countyCourtJudgment,
+            claim.getExternalId(),
+            authorisation,
+            true);
```
Comment (dharmendrak) [inline src/main/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceService.java]: ask question to Robert, if we have to pass false as we are changing ccd by admission to ccd by default?

Comment (sabahirfan) [reply src/main/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceService.java]: That is separate story to take care of this.

**Comment by sabahirfan** [comment]
Another PR on it's way from the same branch.

**Pull request merged**